src/backend/src/prisma/schema.prisma

```diff
@@ -104,6 +104,7 @@ model User {
   reimbursementStatuses     Reimbursement_Status[]
   reimbursements            Reimbursement[]         @relation(name: "purchaser")
   submittedReimbursements   Reimbursement[]         @relation(name: "submitter")
+  teamsAsLead               Team[]                  @relation(name: "teamsAsLead")
```

Comment:
move this up with the other team relations, like just literally cut and paste it so that theyre all together